content/tutorials/atmos-getting-started.md

```diff
@@ -0,0 +1,270 @@
+---
+title: "Getting started with Atmos"
+description: "Learn what Atmos is and how you can start using it with stacks to simplify your DevOps Automation tasks."
+weight: 2
+---
+
+## Intro
+
+`atmos` is a SweetOps built tool to make DevOps and Cloud automation easy. It has direct support for automating Terraform, Helm, Helmfile, and Istio. By natively utilizing [stacks]({{< relref "fundamentals/concepts.md#stacks" >}}), `atmos` enables you to effortlessly manage your Terraform and Helmfile [components]({{< relref "fundamentals/concepts.md#components" >}}) from your local machine or in your CI / CD pipelines.
+
+In this tutorial we'll be looking at a simple (albeit contrived) example of automating multiple Terraform components together into a workflow. This will give you some understanding of what `atmos` can do while also giving you some experience with using it at the command line.
+
+## Prerequisites
+
+### Requirements
+
+To accomplish this tutorial, you'll need to have [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) and [Docker](https://docs.docker.com/get-docker/) installed on your local machine. **That's all**.
+
+### Understanding
+
+Prior to starting this tutorial, you should be sure you understand [our various concepts and terminology]({{< relref "fundamentals/concepts.md" >}}) and have gone through [Getting started with Geodesic]({{< relref "tutorials/geodesic-getting-started.md" >}}) as we'll be using Geodesic as our means to run `atmos`.
```

Comment:
```suggestion
Prior to starting this tutorial, you should be sure you understand [our various concepts and terminology]({{< relref "fundamentals/concepts.md" >}}) and have gone through our [Getting started with Geodesic]({{< relref "tutorials/geodesic-getting-started.md" >}}) tutorial because we'll be using Geodesic as our means to run `atmos`.
```